docs/getting_started.rst

```diff
@@ -58,7 +58,7 @@ We recommend creating an entrypoint file for executing the bot, which will look
         bot.run()
 
 You can then set the executable bit on this file and run the bot straight from the command line.
-For more information on configuring bot settings and plugins, please see the respective :doc:`settings` :doc:`plugins` pages.
+For more information on configuring bot settings and plugins, please see the respective :ref:`settings <settings>` and :ref:`plugins <plugins>` pages.
```

Comment:
Not having a fun time with rst :disappointed:. Whats the syntax to link to an internal page?

Reply:
I realised this was just Github failing to render the file properly. It works in readthedocs.